ui/src/app/components/cromwell-configuration-panel.spec.tsx

```diff
@@ -101,19 +101,26 @@ describe('CromwellConfigurationPanel', () => {
   it('start button should create cromwell and close panel', async () => {
     jest
       .spyOn(appsApi(), 'listAppsInWorkspace')
-      .mockImplementationOnce(() => Promise.resolve([]));
+      .mockImplementation((): Promise<any> => Promise.resolve([]));
```

Comment:
why this change?

Reply:
This was accidental. Thanks for bringing it to my attention.